set_up_authorized_keys.rb

```diff
@@ -10,12 +10,16 @@
 gh_users = (ENV['GITHUB_USER'] || '').split(',')
 
 gh_keys = if gh_users.size > 0
-  gh_users.map { |u|
-    open("https://github.com/#{u}.keys").read
-  }.join("\n")
-else
-  nil
-end
+            gh_users.map do |u|
+              key_url = "https://github.com/#{u}.keys"
+              begin
+                open(key_url).read
+              rescue OpenURI::HTTPError => e
+                puts "[WARN]: #{e.message} while opening #{key_url}, #{u}'s key is ignored'"
```

Comment:
[nits] Use `$stderr.puts` instead.

Reply:
thx for the review. Fixed in cd56948.
Some tool in my editor replaces `$stderr.puts` with `warn`